### ISSUE TITLE
Accessibility issues for radio and checkbox

### ISSUE THREAD

**Issue by map612** [vmware/clarity#4437]
Accessibility check is required for my new project, and accessibility issue detected as below,
![image](https://user-images.githubusercontent.com/7401307/78338748-eb9a8180-75c5-11ea-885a-07cf2bb6a757.png)
And there's detail description from the tool below,

> **What It Means**
> An aria-labelledby or aria-describedby reference exists, but the target for the reference does not exist.
> **Why It Matters**
> ARIA labels and descriptions will not be presented if the element referenced does not exist in the page.
> **How to Fix It**
> Ensure the element referenced in the aria-labelledby or aria-describedby attribute value is present within the page and presents a proper label or description.

I would like to know if this is an issue or not for the Clarify project.

**Comment by gnomeontherun** [comment]
Hello @map612 can you provide a sample reproduction of your test here? I don't know if you have a problem with your implementation or if Clarity has something to consider. See https://stackblitz.com/@clr-team for starters.

**Comment by gnomeontherun** [comment]
Closing as cannot reproduce and lacking actionable details.
